scripts/_GameMaker.js

```diff
@@ -2370,6 +2343,26 @@ function GameMaker_Tick()
         g_MouseDeltaY = 0;
     }
     
+
+    // schedule next frame:
+    // this might be best done after if(!Run_Paused) block,
+    // but then an exception would halt the game loop.
+    var nextFrameAt = g_FrameStartTime + 1000 / TargetSpeed;
+    var now = g_CurrentTime;

```

Comment:
Using `g_CurrentTime` instead of `Date.now();` seems to be locking the runner to not go past the first frame of execution.